README.md

```diff
@@ -241,48 +159,28 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+> If you liked this project and would like to support me email me through my email address andohfrancis9187@gmail.com
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+## 🙏 Acknowledgments <a name="acknowledgements">Ishmael</a>
 
-I would like to thank...
+> I would like to thank my Microvese partner who helped me a lot during the project
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
 
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
 
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
 
-## 📝 License <a name="license"></a>
+## 📝 License <a name="license">MIT</a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Great work on including your **LICENSE** in the **README** file! However, it seems that the link is not functioning as intended, possibly because you have a separate MIT.md file in your project. To address this,
I recommend updating the line to accurately reflect the path of your **MIT** license. 👌
For example, you could modify it to: 

  `This project is [MIT](./MIT.md) licensed.`

  By making this adjustment, the link will direct readers to the correct location of the **MIT** license in your project. Keep up the good work!